internal/resources/cloud/resource_cloud_access_policy.go

```diff
@@ -319,6 +319,9 @@ func flattenCloudAccessPolicyRealm(realm []gcom.AuthAccessPolicyRealmsInner) []i
 }
 
 func flattenCloudAccessPolicyConditions(condition *gcom.AuthAccessPolicyConditions) []interface{} {
+        if condition == nil || len(condition.GetAllowedSubnets()) == 0 {
```

Comment:
Linter is failing in this line since the spaces at the beginning. Could you fix that?

Reply:
Done @spinillos